pmm-app/tests/upgradePMM_test.js

```diff
@@ -0,0 +1,95 @@
+const assert = require('assert');
```

Comment:
4-space indenting applied to this file, unlike the re rest of the app, where 2-space indenting is applied. Let's be consistent. I assume the IDE should support `.editorconfig` via a plugin or something )

Reply:
done, @atymchuk  could you please update the .editorconfig file with tests/ included ?